wordpress-mobile/WordPress-iOS
# Stats Revamp: Remove the customise card from Insights

This PR removes the customise card from the Stats Insights screen

Fixes #18051

To test:

Test as a user who has already installed the app
1. Install the app on simulator or device from trunk (or if you had already installed the WPiOS app from a diff branch, that should suffice
2. Go to the stats screen and click insights tab for a large site such as Field Guide.  Customise card should appear
3. Install the app on the same simulator or device from this branch / PR
4. Go to the stats screen and click insights for a large site such as Field Guide.  Customise card should not appear


Test as a fresh install.
1. Delete the app on simulator / device
2. Launch app
3. Go to a large site such as Field Guide, then go to Stats screen and then click Insights tab
4. No customise card should appear

## Regression Notes
1. Potential unintended areas of impact2. 
Stats insights screen will continue to show customise or grow audience cards

3. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested on simulator and wrote a unit test for headless functionality

4. What automated tests I added (or what prevented me from doing so)
Unit test for SiteStatsInformation

PR submission checklist:
